### PR TITLE
feat: added eslint-plugin-i18n-json

### PR DESCRIPTION
**Pull request by mansurishmuhametov**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows our guidelines: https://github.com/ngxs/store/blob/master/CONTRIBUTING.md#commit
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,6 +24,7 @@
         "format": "yarn package:sort && yarn pretty:format --write",
         "postinstall": "ngcc --async && husky install && lerna run symlink",
         "lint": "yarn eslint \"**/*.ts\" --ignore-pattern **/file-suites/**",
+        "lint:i18n": "yarn eslint --format node_modules/eslint-plugin-i18n-json/formatter.js **/i18n/**/*.json",
```
Comment (Schirbak) [inline package.json:27]: did you forgot to remove this line?

File: packages/eslint-config/index.js [hunk 2]
```diff
@@ -539,6 +539,19 @@ module.exports = {
         {
             files: ['*.component.ts'],
             extends: ['plugin:@angular-eslint/template/process-inline-templates']
+        },
+        {
+            files: ['**/i18n/**/*.json'],
+            extends: ['plugin:i18n-json/recommended'],
+            rules: {
+                'i18n-json/valid-message-syntax': 0,
+                'i18n-json/sorted-keys': [
```
Comment (Schirbak) [inline packages/eslint-config/index.js:548]: can you put it in one line?

**Review by Schirbak — CHANGES_REQUESTED** [review]
most of our eslint rules are covered with tests. i see you added some test files in `integration` folder. but you should pack it more deeper — in the file-suites folder and update tests with new error messages.

**Review by splincode — DISMISSED** [review]

File: packages/eslint-config/index.js (hunk 2 above)
Comment (mansurishmuhametov) [reply packages/eslint-config/index.js]: sure

**Review by splincode — APPROVED** [review]

**Review by Schirbak — APPROVED** [review]

**Pull request merged**